packages/node_modules/@ciscospark/react-component-activity-menu/src/index.js

```diff
@@ -2,71 +2,43 @@ import React, {PropTypes} from 'react';
 import classNames from 'classnames';
 
 import Button from '@ciscospark/react-component-button';
-import {ICON_TYPE_EXIT, ICON_TYPE_MESSAGE, ICON_TYPE_VIDEO_OUTLINE} from '@ciscospark/react-component-icon';
+import ButtonControls from '@ciscospark/react-component-button-controls';
+import {ICON_TYPE_EXIT} from '@ciscospark/react-component-icon';
 
 import styles from './styles.css';
 
 export default function ActivityMenu(props) {
   const {
-    onExit,
-    onMeet,
-    onMessage
+    buttons,
+    onExit
   } = props;
 
   function handleExit(e) {
     e.preventDefault();
     onExit();
   }
 
-  function handleMeet(e) {
-    e.preventDefault();
-    onMeet();
-  }
-
-  function handleMessage(e) {
-    e.preventDefault();
-    onMessage();
-  }
-
   return (
     <div className={classNames(`activity-menu`, styles.activityMenu)}>
-      <div className={classNames(`activity-menu-exit`, styles.exit)}>
-        <div className={classNames(`menu-item`, styles.menuItem)}>
-          <div className={classNames(`button-container`, styles.buttonContainer)}>
-            <div className={classNames(`button-item`, styles.buttonItem)}>
-              <Button iconType={ICON_TYPE_EXIT} onClick={handleExit} title={`Exit`} />
-            </div>
-          </div>
+      {
+        onExit &&
```

Comment:
Oops, missed the required field. I think the best approach is to remove the isRequired, so that  in a case where the exit button isn't needed, we don't have to show it.